aptos-move/block-executor/src/view.rs

```diff
@@ -219,6 +243,58 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         }
         ret
     }
+
+    #[allow(unused)]
+    /// Same as `get_base_value`, but also tries to lift values.
+    fn get_base_value_with_layout(
+        &self,
+        layout: &MoveTypeLayout,
+        state_key: &T::Key,
+    ) -> anyhow::Result<Option<StateValue>> {
+        // Callback for replacing values with identifiers. Right now it is
+        // implemented as a serialization roundtrip.
+        let process_lifting = |bytes: Vec<u8>| -> anyhow::Result<Vec<u8>> {
+            // TODO: alert on fail?
```

Comment:
error!() is alert.

if it is fine in speculative execution, but not in final, then I think speculative_error!() is the macro. @gelash